osutil/mountentry.go

```diff
@@ -378,6 +378,25 @@ func (e *MountEntry) XSnapdDetach() bool {
 	return e.OptBool("x-snapd.detach")
 }
 
+// XSnapdSymlink returns the target for a symlink mount entry.
+//
+// For non-symlinks an empty string is returned.
+func (e *MountEntry) XSnapdSymlink() string {
+	val, _ := e.OptStr("x-snapd.symlink")
+	return val
+}
+
+// XSnapdIgnoreMissing returns true if a mount entry should be ignored
+// if the source or target are missing.
+//
+// By default, snap-update-ns will try to create missing source and
+// target paths when processing a mount entry.  In some cases, this
```

Comment:
Double space between '.' and 'In'.

Reply:
Many text editors use that as a default.